### PR TITLE
Update prerequisites with scraped data

### PR DESCRIPTION
**Pull request by EmilioCristalli**

### CODE REVIEW

File: db/data/scraped_prerequisites.yml [hunk 1]
```diff
@@ -9139,16 +9146,16 @@ prerequisites:
...
-  - type: subject
-    needs: course
-    subject_needed_code: '1023'
-    subject_needed_name: MATEMATICA DISCRETA 1
+      subject_needed_code: '1023'
+      subject_needed_name: MATEMATICA DISCRETA 1
+      needs: course
+    - type: subject
+      subject_needed_code: SRN14
+      subject_needed_name: SRN14 - MATEMÁTICA DISCRETA I
```
Comment (santiagorodriguez96) [inline db/data/scraped_prerequisites.yml:9157]: Is it correct that the code shows up in the subject name? Did we break the scrapper?

**Review by santiagorodriguez96 — COMMENTED** [review]
So now `SRN14 - MATEMÁTICA DISCRETA I` is also an option for the course of `1027 - LOGICA` as well as `1023 - MATEMATICA DISCRETA 1`. Is that correct?

File: db/data/scraped_prerequisites.yml (hunk 1 above)
Comment (EmilioCristalli) [reply db/data/scraped_prerequisites.yml:9157]: afaict that's a pre-existing issue: https://github.com/cedarcode/mi_carrera/blob/ad0ae618cb6ccc6661a1e499ddafeb919a6363a1/db/data/scraped_prerequisites.yml#L9770

and will be fixed in the scraper refactor PR: https://github.com/cedarcode/mi_carrera/pull/235/files#diff-0a345fa99c9566f09a88e31e8abda878c16cfd362f5456548c60edc20c498997R9770

**Comment by EmilioCristalli** [comment]
> So now `SRN14 - MATEMÁTICA DISCRETA I` is also an option for the course of `1027 - LOGICA` as well as `1023 - MATEMATICA DISCRETA 1`. Is that correct?

correct

image

File: db/data/scraped_prerequisites.yml (hunk 1 above)
Comment (santiagorodriguez96) [reply db/data/scraped_prerequisites.yml:9157]: Good! Interesting 🤔

**Review by santiagorodriguez96 — APPROVED** [review]
LGTM :shipit:

**Pull request merged**